google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/st/STBigQueryStorageLongRunningWriteTest.java

```diff
@@ -0,0 +1,211 @@
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.DatasetInfo;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.StandardSQLTypeName;
+import com.google.cloud.bigquery.StandardTableDefinition;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableInfo;
+import com.google.cloud.bigquery.TableResult;
+import com.google.cloud.bigquery.storage.v1beta2.AppendRowsResponse;
+import com.google.cloud.bigquery.storage.v1beta2.BigQueryWriteClient;
+import com.google.cloud.bigquery.storage.v1beta2.JsonStreamWriter;
+import com.google.cloud.bigquery.storage.v1beta2.StreamWriter;
+import com.google.cloud.bigquery.storage.v1beta2.TableName;
+import com.google.cloud.bigquery.storage.v1beta2.it.ITBigQueryStorageLongRunningTest;
+import com.google.cloud.bigquery.testing.RemoteBigQueryHelper;
+import com.google.protobuf.Descriptors;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Logger;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.threeten.bp.Duration;
+import org.threeten.bp.LocalDateTime;
+
+public class STBigQueryStorageLongRunningWriteTest {
+  private static final Logger LOG =
+      Logger.getLogger(ITBigQueryStorageLongRunningTest.class.getName());
+
+  private static BigQueryWriteClient client;
+  private static String parentProjectId;
+  private static final String DATASET = RemoteBigQueryHelper.generateDatasetName();
+  private static final String TABLE = "testtable";
+  private static final String DESCRIPTION = "BigQuery Write Java long test dataset";
+
+  private static TableInfo tableInfo;
+  private static String tableId;
+  private static BigQuery bigquery;
+
+  private static JSONObject MakeJsonObject(int size) throws IOException {
+    JSONObject object = new JSONObject();
+    // size: (1, simple)(2,complex)()
+    if (size == 1) {
+      object.put("test_str", "aaa");
+      object.put("test_numerics", new JSONArray(new String[]{"1234", "-900000"}));
+      object.put("test_datetime", String.valueOf(LocalDateTime.now()));
+    }
+    else if (size == 2) {  // make it complicated and large
+      // TODO(jstocklass): Make a better json object that doesn't break the format rules.
+    }
+    return object;
+  }
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    //Assume.assumeTrue(LONG_TESTS_DISABLED_MESSAGE, Boolean.getBoolean(LONG_TESTS_ENABLED_PROPERTY));
+    client = BigQueryWriteClient.create();
```

Comment:
```
2021-03-02T00:42:09.3881890Z [ERROR] Errors: 
2021-03-02T00:42:09.3884743Z [ERROR]   STBigQueryStorageLongRunningWriteTest.beforeClass:87 » IO The Application Defa...
```
[The error](https://pipelines.actions.githubusercontent.com/QcNa0RdPyA3gi7GiY9vdwe6CZOh44p39bM3R2oEqvGycZ0S9TC/_apis/pipelines/1/runs/3895/signedlogcontent/15?urlExpires=2021-03-02T17%3A30%3A36.7452972Z&urlSigningMethod=HMACV1&urlSignature=g0pEWM%2FS%2FFIUbILOPJ5mWVoVKsPRLdgYxE35nrIp97Q%3D) is on this line.

Reply:
Yeah it's giving me the old error, it seems like it's trying to use the cloudtop-prod again, but only in maven and on github. Not sure what to change to get rid of this error.

Reply:
The issue was the naming of the test being ST instead of IT